static/app/components/events/eventEntries.tsx

```diff
@@ -489,10 +489,16 @@ function MiniReplayView({
 }: MiniReplayViewProps) {
   const hasEventAttachmentsFeature = orgFeatures.includes('event-attachments');
   const hasSessionReplayFeature = orgFeatures.includes('session-replay-ui');
+  const errorEventTimestamp = event.dateCreated;
 
   if (replayId && hasSessionReplayFeature) {
     return (
-      <EventReplay replayId={replayId} orgSlug={orgSlug} projectSlug={projectSlug} />
+      <EventReplay
+        replayId={replayId}
+        orgSlug={orgSlug}
+        projectSlug={projectSlug}
+        eventTimestamp={errorEventTimestamp}
```

Comment:
the rrwebintegration below passes the whole event: `<RRWebIntegration
        event={event}` would that be easier here too?